src/unsafe_removed_from_name.rs

```diff
@@ -0,0 +1,90 @@
+use rustc::hir::*;
+use rustc::lint::*;
+use syntax::ast::{Name, NodeId};
+use syntax::codemap::Span;
+use syntax::parse::token::InternedString;
+use utils::span_lint;
+
+/// **What it does:** This lint checks for imports that remove "unsafe" from an item's name
+///
+/// **Why is this bad?** Renaming makes it less clear which traits and structures are unsafe.
+///
+/// **Known problems:** None.
+///
+/// **Example:**
+/// ```rust,ignore
+/// use std::cell::{UnsafeCell as TotallySafeCell};
+///
+/// extern crate crossbeam;
+/// use crossbeam::{spawn_unsafe as spawn};
+/// ```
+declare_lint! {
+    pub UNSAFE_REMOVED_FROM_NAME,
+    Warn,
+    "unsafe removed from name"
+}
+
+pub struct UnsafeNameRemoval;
+
+impl LintPass for UnsafeNameRemoval {
+    fn get_lints(&self) -> LintArray {
+        lint_array!(UNSAFE_REMOVED_FROM_NAME)
+    }
+}
+
+impl LateLintPass for UnsafeNameRemoval {
+    fn check_mod(&mut self, cx: &LateContext, m: &Mod, _: Span, _: NodeId) {
+        // only check top level `use` statements
```

Comment:
Thanks! I'll totally do that. I did it the way I did because I was basing it off of the glob_enum lint to get an idea of how to lint `use` statements.